### PR TITLE
Textlabel properties

### PR DESCRIPTION
**Pull request by ckdavid**

### CODE REVIEW

File: lib/Enum/TextTruncate.lua [hunk 1]
```diff
@@ -0,0 +1,6 @@
+local createEnum = import("../createEnum")
+
+return createEnum("TextTruncate", {
+    None = 0,
+    AtEnd = 1,
```
Comment (LPGhatguy) [inline lib/Enum/TextTruncate.lua]: This file is indented with spaces -- the standard for Lemur and Roblox codebases is tabs!

You can check out our style guide (https://roblox.github.io/lua-style-guide/) for other details. We opened it up to the public super recently but haven't quite announced it yet.

**Comment by ckdavid** [comment]
Do you use an auto-formatter? If so, which one? It would make sense for us
to be using the same!

**Comment by Kampfkarren** [comment]
VS Code with the option to remove trailing whitespace on save works fine for me.

**Comment by ckdavid** [comment]
So you aren't using any VS Code extensions internally to ensure consistent
formatting of all your Lua code?

**Comment by ckdavid** [comment]
Sorry, I didn't look at who I'm replying to! I was wondering whether Roblox
use any extensions internally.

**Comment by LPGhatguy** [comment]
There aren't very many (any?) good Lua autoformatting tools. We're working on some (a hacky short-term one and and a good long-term one) internally, but they're still a little ways out.

I enable trailing whitespace trimming in my editor (Sublime Text) as well as visible whitespace. It's much easier to catch problems that way!

**Pull request merged**